browse/templates/archive/single_archive.html

```diff
@@ -0,0 +1,89 @@
+{%- extends "base.html" -%}
+
+{% block head %}
+{{ super() }}
+<link rel="alternate" type="application/rss+xml" title="{{archive['name']}}" href="/rss/{{archive_id}}"/>
+{% endblock head %}
+
+{% block title %}{{archive.name}}{% endblock %}
+
+{% block header_h1 %}
+<h1>
+  > <a href="{{url_for('.list_articles', context=archive_id, subcontext='recent')}}">{{archive_id}}</a>
+</h1>
+{% endblock %}
+
+
+{%- block content %}
+<div id='content'>
+  <h1>{{archive.name}} <span style="font-size: 70%;">(since {{archive.start_date}})</span></h1>
```

Comment:
Nitpicking here, but we should probably preserve the month granularity from classic in the display, since I think we're defaulting to the first day of each month for all the `start_date` values (mainly because we can't set an undefined day in `datetime`).

This would be fine:
``{{ archive.start_date.strftime('%B %Y') }}``